server/testing/src/main/java/org/apache/james/junit/categories/Unstable.java

```diff
@@ -0,0 +1,5 @@
+package org.apache.james.junit.categories;
```

Comment:
We are missing the compulsary Apache v2 license :-/

Reply:
yes, I'm surprised this is not enforced by the build :)  but this is a test I'm having issues configuring some of the modules 
```
	modified:   server/protocols/jmap-draft-integration-testing/cassandra-jmap-draft-integration-testing/pom.xml
	modified:   server/protocols/jmap-draft-integration-testing/rabbitmq-jmap-draft-integration-testing/pom.xml
	modified:   server/protocols/jmap-rfc-8621-integration-tests/distributed-jmap-rfc-8621-integration-tests/pom.xml
	modified:   server/protocols/webadmin-integration-test/distributed-webadmin-integration-test/pom.xml
```
but I wanted to test the parallel stage setup but it's not working out so great :( https://builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-268/3/pipeline